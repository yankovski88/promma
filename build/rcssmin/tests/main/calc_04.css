/*! Found this example by searching github for 'calc(' :-)
 * https://github.com/marcelotorres/experiencing-transition-and-the-calc-CSS3/blob/master/style.css
 */
.mtlab1-wrap{margin:60px auto 0 auto;overflow:hidden;background-color:#f0f0f0;border:2px solid #fff;
			width: -webkit-calc((74px * 6) + 330px);
			width: -moz-calc((74px * 6) + 330px);
			width: -o-calc((74px * 6) + 330px);
			width: calc((74px * 6) + 330px);
}
