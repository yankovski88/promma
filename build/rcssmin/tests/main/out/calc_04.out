.mtlab1-wrap{margin:60px auto 0 auto;overflow:hidden;background-color:#f0f0f0;border:2px solid #fff;width:-webkit-calc((74px * 6) + 330px);width:-moz-calc((74px * 6) + 330px);width:-o-calc((74px * 6) + 330px);width:calc((74px * 6) + 330px)}